### PR TITLE
feat: add additional data capture - CLA Text URL and CLA Text

### LINKED ISSUE #92: FEAT: Record URL to CLA text actually signed in database
Let's record the actual URL behind the text that was signed by someone as well as the the version number (which we already record).

For belts and braces - we can also keep a full copy of the CLA text too.

### PR DESCRIPTION
**Pull request by madpah**
Resolves #92.

Initial PR @bhamail for some 👀 please.

### CODE REVIEW

File: db/migrations/002_cla_url_text.up.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+ALTER TABLE signatures
+    ADD COLUMN ClaTextUrl VARCHAR(250) NOT NULL,
```
Comment (bhamail) [inline db/migrations/002_cla_url_text.up.sql:4]: Cool! Will DB need some default values for any existing records for these? (or just delete 'em)?

**Review by bhamail — APPROVED** [review]
Noice! Some minor questions inline.

File: db/migrations/002_cla_url_text.up.sql (hunk 1 above)
Comment (madpah) [reply db/migrations/002_cla_url_text.up.sql:4]: Good point!

**Comment by madpah** [comment]
Comments addressed @bhamail - appreciate any final thoughts?

**Pull request merged**